index.html

```diff
@@ -272,6 +272,30 @@
     </section>
     <section>
       <h2>
+        Security and privacy considerations
+      </h2>
+      <p>
+        Application of a wake lock causes various device components such as
+        display or CPU to operate at higher power levels than they otherwise
+        would. This can lead to undesirable and potentially dangerous effects
+        such as excessive heating and faster than normal battery charge
+        depletion. The latter is particularly relevant to mobile devices which
+        may not have a stationary power source readily available. Complete
+        battery depletion at an unexpected time can lead to inability of the
+        user to make or receive calls and use network services, including the
+        emergency call service. Implementations should consider preventing
+        wake lock application if they determine that the remaining battery
+        capacity is low.
+      </p>
+      <p>
+        Another concern is that a wake lock can be requested in a nested thrid
+        party context, which means that the top-level context may not be
+        completely in control whether a wake lock can potentially be requested
+        by this particular page if it incorporates third party content.
```

Comment:
I think rather than just documenting that risk, we should prevent it from happening, i.e. require that only a top level browsing context, or nested browsing contexts from the same origin, may request wake lock.